app/helpers/bulk_download_types_helper.rb

```diff
@@ -17,8 +17,9 @@ module BulkDownloadTypesHelper
       display_name: "Combined Sample Taxon Results",
       description: "The value of a particular metric (e.g. total reads, rPM) for all taxons in all selected samples, in a single data table",
       category: "report",
-      options: [
+      fields: [
```

Comment:
Renamed from options to fields, since option is already used to describe the different choices for a field (e.g. ".csv", ".biom") below. `option.options` is super confusing.